Makefile.PL

```diff
@@ -9,63 +9,66 @@
 require 5;
 
 use strict;
-use ExtUtils::MakeMaker;
+use ExtUtils::MakeMaker 6.64;
```

Comment:
could the Makefile.PL updates be moved to a separate commit? Other than adding Sub::Util as a prereq, these changes don't look to be related to the change in prototypes.